integrationtests/src/test/java/org/axonframework/integrationtests/testsuite/administration/MutableReflectionEntityModelAdministrationTest.java

```diff
@@ -18,23 +18,28 @@
 
 import org.axonframework.configuration.Module;
 import org.axonframework.eventsourcing.configuration.EventSourcedEntityModule;
+import org.axonframework.eventsourcing.configuration.EventSourcingConfigurer;
 import org.axonframework.integrationtests.testsuite.administration.common.PersonIdentifier;
 import org.axonframework.integrationtests.testsuite.administration.state.mutable.MutablePerson;
-import org.axonframework.modelling.configuration.StatefulCommandHandlingModule;
+import org.axonframework.commandhandling.configuration.CommandHandlingModule;
 import org.axonframework.modelling.entity.EntityMetamodel;
 
 /**
- * Runs the administration test suite using as many reflection components of the {@link EntityMetamodel} and
- * related classes as possible. As reflection-based components are added, this test may change to use more of them.
+ * Runs the administration test suite using as many reflection components of the {@link EntityMetamodel} and related
+ * classes as possible. As reflection-based components are added, this test may change to use more of them.
  */
 public class MutableReflectionEntityModelAdministrationTest extends AbstractAdministrationTestSuite {
 
+    @Override
+    protected EventSourcingConfigurer testSuiteConfigurer(EventSourcingConfigurer configurer) {
+        var personEntity = EventSourcedEntityModule.annotated(PersonIdentifier.class, MutablePerson.class);
+        return configurer.componentRegistry(cr -> cr.registerModule(personEntity));
+    }
+
     @Override
     Module getModule() {
```

Comment:
So...all this method does is set the name of the `Module`, right? Perhaps we can simplify that? The crux of this test suite lies in specific types of `EntityModules` now.

Reply:
Now that I look better at it, I think we can drop the `getModule` method entirely from the `AbstractAdministrationTestSuite`. Registering a `CommandHandlingComponent` without any command handlers, well, is useless.

Reply:
Ahh... yes! It was just for entities here. 
I'm going to change a little the `CommandHandlingModule` API to force on the compile time at least one command handler, but not in this PR.